addons/purchase_stock/views/res_config_settings_views.xml

```diff
@@ -17,7 +17,6 @@
 						</div>
 						<div class="o_setting_right_pane">
 							<label for="module_stock_dropshipping"/>
-							<a href="https://www.odoo.com/documentation/user/14.0/purchase/replenishment/flows/dropshipping.html" title="Documentation" class="o_doc_link" target="_blank"></a>
```

Comment:
could replace by this one https://www.odoo.com/documentation/user/14.0/inventory/management/delivery/dropshipping.html